pkg/resource/hyper_parameter_tuning_job/testdata/hyper_parameter_tuning_job/v1alpha1/hptj_invalid_before_create.yaml

```diff
@@ -18,8 +18,6 @@ spec:
         minValue: '10'
         maxValue: '20'
         scalingType: Linear
-      continuousParameterRanges: []
```

Comment:
what changed to require this change to testdata or was it redundant to begin with ?

Reply:
Delta now using deepReflect so having this causes a diff which thus causes the unit-test to fail.

Reply:
but we should be handling the diff generated correct instead of changing the unit test ?

Reply:
In general, yes, but this is anyway invalid input so it's fine

Reply:
test_suite_runner.go:144: 
        	Error Trace:	test_suite_runner.go:144
        	            				test_suite_runner.go:115
        	            				test_suite_runner.go:91
        	Error:      	Not equal: 
        	            	expected: 0
        	            	actual  : 2
        	Test:       	TestHyperParameterTuningJobTestSuite/Create=InvalidInput
  ```
Unexpected differences:
  Path: {[Spec HyperParameterTuningJobConfig ParameterRanges CategoricalParameterRanges]}, expected: [], actual: []
  See expected differences below:
  Path: {[Spec HyperParameterTuningJobConfig ParameterRanges ContinuousParameterRanges]}, expected: [], actual: []
```

Reply:
In the before create yaml it has these two fields which are supposed to be lists of dicts/maps however they are empty. In the after yaml they are not included in the spec so, I decided to remove it from the test, can attempt to try to debug, however diff message is confusing.

Reply:
I see, this is fine for now. Better to have a cleaner unit test.